README.md

```diff
@@ -8,7 +8,7 @@
 * [Available Serializers](#available-serializers)
 * [Usage](#usage)
 
-The purpose of this library is to transform search string to a intermediary tokenized data structure that can be used to perform analysis on the search or middle ground to translate the query into other DSL or query structures.
+The purpose of this library is to transform a search string to a intermediary tokenized data structure that can be used to perform analysis on the search input or as an intermediate structure to translate the query into other DSLs or query structures.
```

Comment:
Sorry, there is one more little thing...
It's `a**n** intermediary tokenized`. Sorry if I missed this before.